src/Relude/Base.hs

```diff
@@ -14,11 +14,7 @@ Reexports from @Data.*@ and @GHC.*@ modules of
 
 module Relude.Base
        ( -- * Base types
-         module Data.Bits
-       , module Data.Char
-       , module Data.Int
-       , module Data.Word
-       , Natural
+         module Data.Char
```

Comment:
Heh, section with `Base types` became much smaller... But it's okay to keep it as it is for now to not mess with the imports.